src/locales/en.json

```diff
@@ -23,6 +23,7 @@
   "inventory variance": "inventory variance", 
   "Instance Url": "Instance Url",
   "Item added to upload list": "Item added to upload list",
+  "Item removed successfully":"Item removed successfully",
```

Comment:
Update the translation as this text is changed to `Item removed from upload list`